examples/websocket-relay/relay-app/src/components/WorkflowDetail.svelte

```diff
@@ -1,49 +1,50 @@
 <script lang="ts">
-  import { JsonView } from "@zerodevx/svelte-json-view";
-  import { slide } from "svelte/transition";
-  import { quartOut } from "svelte/easing";
+    // import { JsonView } from "@zerodevx/svelte-json-view";
+    import { slide } from "svelte/transition";
```

Comment:
yikes! I didn't actually change this file, so I think there's some tabs vs spaces stuff going on. let me take a look

Reply:
ahh looks like it was modified in the previous commit before I added my changes(same with the `TaskValue` file). I'll see if I can clear out those spacing changes https://github.com/ipvm-wg/homestar/pull/358/commits/e9969288d0c183f0586038ac9f2d0e03bca5b806